### ISSUE TITLE
Prison labels showing too late

### ISSUE THREAD

**Issue by dieterdreist** [gravitystorm/openstreetmap-carto#4875]
### Expected behavior
have the prison icon and label visible at zoomlevels where the prison is already well visible
Prison labels should also have precedence over road labels

### Actual behavior
minor pois are shown earlier and the prison label and icon only from z17, it would be more reasonable to show an icon in z13 or 14 and a label in z14/15. 

### Screenshots with links illustrating the problem
from https://www.openstreetmap.org/#map=15/52.3367/13.2782

z13_prison:
![z13_prison](https://github.com/gravitystorm/openstreetmap-carto/assets/581288/f36956b0-a5d2-4f52-91d4-642d8f570030)

z14_prison:
![z14_prison](https://github.com/gravitystorm/openstreetmap-carto/assets/581288/cc8d1f99-6e60-4881-bf4f-91843eab5699)

z15_prison:
![z15_prison](https://github.com/gravitystorm/openstreetmap-carto/assets/581288/df5dc9bc-9347-4cb3-8646-be17eb556f35)

z16_prison:
![z16_prison](https://github.com/gravitystorm/openstreetmap-carto/assets/581288/31283a0c-fb2d-4f4f-ae1f-868ed583b839)

z17_prison:
![z17_prison](https://github.com/gravitystorm/openstreetmap-carto/assets/581288/de1009ba-afa9-4841-850b-01d0adaa2358)

Here is another example, significantly bigger, even at z12 it is well visible:
https://www.openstreetmap.org/#map=12/41.9347/12.5749

**Comment by dch0ph** [comment]
I agree that the symbol + label are arriving rather late. This is clear at Z16, where the symbol for the prison library is appearing before the symbol for the prison itself.

IIRC, the only symbols that appear as early as Z13 are wilderness/alpine huts (for obvious reasons). 

Better comparisons would be:
schools: No school symbol, but with landcover-style (italic) labelling as soon as critical size is reached (3000 pixels). 
hospitals: Symbol appears at Z15, label from Z16.
These both potentially cover large areas with multiple buildings.

Personally I think the school mapping is an oddity (I notice Tracestack Carto has introduced a school symbol), and the hospital approach would work. Given that prisons will be generally quite large (and not common), we don't need to worry about a symbol at Z15 crowding the map.

Some prisons will already be quite visible by Z14, but this is where the landcover shading is doing its job.

**Comment by dieterdreist** [comment]
sent from a phone

> On 26 Dec 2023, at 18:31, dch0ph ***@***.***> wrote:
> 
> Some prisons will already be quite visible by Z14, but this is where the landcover shading is doing its job.
> 


prisons are often much bigger than schools, and we tend to map them both as areas so we could see when they are “already quite visibile” and show an icon and or a label in these cases
